### ISSUE TITLE
Unknown language: `markdown` is not registered

### ISSUE THREAD

**Issue by jalehman** [tloncorp/landscape-apps#2121]
Can't read a post in a notebook that's written in markdown. 

```
Error: Unknown language: `markdown` is not registered
    at b1e.yxt [as highlight] (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:1150:253330)
    at https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:1151:8362
    at gR (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:93:53252)
    at JG (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:93:62788)
    at pae (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:93:62114)
    at bX (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:93:61697)
    at Nae (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:97:12048)
    at Tae (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:97:964)
    at DSe (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:97:887)
    at Lw (https://wolref-podlex.arvo.network/apps/groups/assets/index.f67c0ed6.js:97:737)
```

**Comment by jamesacklin** [comment]
I'm not in this group so I can't tell what's going on, but I have a feeling it has to do with the syntax highlighting in the code block.

**Comment by jamesacklin** [comment]
@belisaruis222 Ping? Getting a lot of feedback on this one.

**Comment by belisarius222** [comment]
I copy-pasted the raw text of a github gist written in markdown into a blank new post.  If I remember right, it said "plain text" in the dropdown, so I switched it to markdown.  Then syntax highlighting appeared, but the markdown did not render into rich text.

The post loads fine for me, although I couldn't figure out how to get the markdown rendered or get long lines to wrap.

**Comment by jamesacklin** [comment]
Thank you. URL of the gist? You used the Groups app?

**Comment by belisarius222** [comment]
Yes, I used the groups app.  It was a private gist, but I copied raw text in markdown format from github's "raw" page, just like this one:
https://gist.githubusercontent.com/belisarius222/a05fb9b688dada102be9020bb9b429d1/raw/6bb5ea5c7a165047da8141f4f53274faf072824a/libames.md

**Comment by jamesacklin** [comment]
Cool, thanks.

Hilariously enough, when we investigated this, we found that the "raw" page is just a HTML <pre> tag with the file contents spat into it; it's not actually serving plain text. This explains why your pasting into a Notebook triggered the code block detection.

We fixed the underlying error, but... sneaky!

(Command + Shift + P will strip formatting on paste and do what you expected.)
